test/e2e/egress_test.go

```diff
@@ -601,6 +601,13 @@ func testEgressNodeFailure(t *testing.T, data *TestData) {
 			signalAgent := func(nodeName, signal string) {
 				cmd := fmt.Sprintf("pkill -%s antrea-agent", signal)
 				rc, stdout, stderr, err := data.RunCommandOnNode(nodeName, cmd)
+				// Try sudo if the pkill operation is not permitted
```

Comment:
Why is this change required? it didn't need sudo on ubuntu-1804 but need it on ubuntu-2004? or it's because different users being used.

There is a method "RunCommandOnNodeExt" that can set sudo, it could use it on first attemp?

Reply:
The reason we need to use root privilege in new image is because pkill updated to 3.3.16 in ubuntu20.04. There is a bug fix in new version that return 0 only if the process is killed successfully. That means the successful result is false positive in previous ubuntu image as well. To fix this bug I have another PR #3948 to fix this issue. After it is merged I will continue to update this PR. @tnqn

Reply:
Thanks. It means the verficiation was wrong. I checked the code and it was indeed wrong. I created #3953, whose verification on jenkins should fail. If it fails as expected, perhaps you could include that commit to https://github.com/antrea-io/antrea/pull/3948.